test/profiling_plonk_circuit.hpp

```diff
@@ -1,222 +0,0 @@
-//---------------------------------------------------------------------------//
```

Comment:
include/nil/blueprint/utils/profiling_plonk_circuit.hpp was deleted in the master branch, so this test makes no sense, and is not used anywhere.